rasa/core/constants.py

```diff
@@ -1,8 +1,10 @@
 DEFAULT_SERVER_PORT = 5005
 
-DEFAULT_SERVER_FORMAT = "{}://localhost:{}"
+DEFAULT_SERVER_INTERFACE = "0.0.0.0"
```

Comment:
hmm so this was `localhost` before, but we actually set the listen host to `0.0.0.0` - which does seem like it was wrong. However I'm slightly worried it may have consequence as we use this for things other than just logging. I _think_ it's most likely fine though...

Reply:
Well, it was `localhost`-in-logging-line-only. Code showed it was actually listening on `0.0.0.0`

Reply:
> 
> 
> Thanks for changing the approach so quickly. 🚀
> 
> Good point about the inconsistency with the SDK, I hadn't realised that - but I think it's probably fine, we can add a similar flag to the action server at some point.
> 
> It seems like you may have to rebase your branch, as some unrelated changes leaking in.

I've been trying to rebase this branch with main all evening and I've not been able to get it done, mainly due to my understanding of git and the vscode tooling I'm using. Any way to get this done without rebasing, @joejuzl? 

> 
> Also could you update the tests at `tests/cli/test_rasa_run.py`?

I'm working on a PR for rasa-sdk which supports both an option and the environment variable, since I contributed that.

Reply:
> Well, it was localhost-in-logging-line-only. Code showed it was actually listening on 0.0.0.0

`DEFAULT_SERVER_FORMAT` is used in `DEFAULT_SERVER_URL` which is used in a couple other places. But I don't think it will cause an issue.

> I've been trying to rebase this branch with main all evening and I've not been able to get it done, mainly due to my understanding of git and the vscode tooling I'm using. Any way to get this done without rebasing, @joejuzl?

I think the problem is the commit `Merge remote-tracking branch 'upstream/2.8.x' into bastbnl-issue-8469`. To fix this you either have to do an interactive rebase onto `main` and skip that commit. Or you can merge in `origin/main` and then `git reset --soft origin/main` (replacing `origin` with whatever the remote called for you) - this will leave all your changes staged but uncommited, on top of master, but you will loose your commit history.